pkg/kots/lint_test.go

```diff
@@ -4767,3 +4769,109 @@ spec:
 		})
 	}
 }
+
+func Test_lintRenderedFilesYAMLValidity(t *testing.T) {
+	type args struct {
+		renderedFiles SpecFiles
+	}
+	tests := []struct {
+		name string
+		args args
+		want []LintExpression
+	}{
+		{
+			name: "valid",
+			args: args{
+				renderedFiles: SpecFiles{
+					{
+						Name:    "valid.yaml",
+						Path:    "valid.yaml",
+						Content: "apiVersion: v1\nkind: Pod\nmetadata:\n  name: valid",
+					},
+				},
+			},
+			want: nil,
+		},
+		{
+			name: "invalid content at line 1",
+			args: args{
+				renderedFiles: SpecFiles{
+					{
+						Name:    "invalid.yaml",
+						Path:    "invalid.yaml",
+						Content: `apiVersion: ***HIDDEN***/0`,
+					},
+				},
+			},
+			want: []LintExpression{
+				{
+					Rule:    "invalid-rendered-yaml",
+					Type:    "error",
+					Path:    "invalid.yaml",
+					Message: "yaml: did not find expected alphabetic or numeric character",
```

Comment:
why does this one not return the line that is invalid in the message? is it an issue with it being in line 1?

Reply:
the module's `yaml.unmarshall` returns the error in that way. so, wanted to add the test with edge case[in future if module code error handling changes, we can update our code accordingly]

Reply:
gotcha, so yaml library does not return a line number.  this is at least better than nothing, so i'm ok with it